styles.css

```diff
@@ -35,47 +35,89 @@ body {
     font-family: var(--font-family-accent);
 }
 
-#destinations,
-#tips {
+.section-destinations,
```

Comment:
I suggest styling all sections at the same time. This is what I have in my code:
 section {
    height: 80vh;
    box-sizing: border-box;
    max-width: var(--max-width-desktop);
}
section:nth-child(2n) {
    background-color: var(--color-isabelline);
}

Reply:
I agree, except Home section should be like this:

section:first-child {
height: 100vh;
}